utilities/cli/holohub.py

```diff
@@ -58,7 +42,7 @@ class HoloHubCLI:
     DEFAULT_SDK_DIR = "/opt/nvidia/holoscan/lib"
 
     def __init__(self):
-        self.script_name = os.environ.get("HOLOHUB_CMD_NAME", "holohub")
+        self.script_name = os.environ.get("HOLOHUB_CMD_NAME", "./holohub")
```

Comment:
Did previous "holohub" without "./" work, or resulted in an error?

Reply:
Ack. Thanks for fixing.

Reply:
I guess CLI tests did not cover this case -- can we update them accordingly? Can be in a separate PR to avoid blocking this one. https://github.com/nvidia-holoscan/holohub/actions/runs/15535221215/job/43732552101